package-lock.json

```diff
@@ -0,0 +1,1092 @@
+{
```

Comment:
what did you add to this file? i didn't add it to the commit because when we push to the heroku app it works without this file, but when he added it to the commit in the tutorial it caused problems

Reply:
This is generated with every npm install so we can omit it since it causes problems

Reply:
i think we should omit this, we can run npm install on our own machine